brainstormforce/astra
# AST-1618 - border color synced to #ddd as per Astra customizer family

### Description
- Border colors updated for customizer controls
- ast-select should have ddd color
- selector active button has fade border, so it should be active BG color one.

### Screenshots
- Updated screenshot - https://share.bsf.io/8LuDlNl7

### Types of changes
- UI chnage (non-breaking change )

### How has this been tested?
- Check any select dropdown in customizer
- Check font-families select2 
- Check button active option border color

### Checklist:
